.github/workflows/python-linters.yml

```diff
@@ -38,16 +69,33 @@ jobs:
       run: |
         git config --global user.email "pypa-dev@googlegroups.com"
         git config --global user.name "pip"
-    - name: Update setuptools and tox dependencies
-      run: |
+    - name: Update setuptools
+      run: >-
         python -m pip install --upgrade setuptools
+    - name: Install tox
+      run: >-
         python -m pip install --upgrade tox tox-venv
+    - name: Log the list of packages
+      run: >-
         python -m pip freeze --all
```

Comment:
nit: These as a single bundle was probably nicer -- it's not really a cause for concern, but I don't think this information has ever been useful for most tasks and even when it is, the existing single bundle is named well enough to know where to look.

Reply:
I think it's mostly personal preference. But I've found out if there's separate distinct steps, it's easier to inspect logs.